README.md

```diff
@@ -13,53 +13,55 @@ See the [Secure Supply Chain docs in Kubewarden](https://docs.kubewarden.io/dist
 
 ## Settings
 
-The policy takes a list of signatures. A signature can be of two types: public key or keyless. Each signature
+The policy takes a list of signatures. A signature can be of four types: GitHub actions, public key, keyless exact match or keyless prefix. Each signature
 has an `image` field which will be used to select the matching containers in the pod that will be evaluated.
 `image` supports wildcard. For example, `ghcr.io/kubewarden/*` will match all images from the kubewarden ghcr repo.
 
-Example:
+Signature types:
 
-```yaml
-modifyImagesWithDigest: true #optional. default is true
-signatures:
-  - image: "*"
-    pubKeys: 
-      - ....
-    annotations: #optional
-      env: prod
-  - image: "ghcr.io/kubewarden/*" 
-    keyless:
-      - issuer: "https://token.actions.githubusercontent.com"
-        subject: "kubewarden"
-    annotations: #optional
-      env: prod
-```
-
-This policy will validate all images with the public keys provided, and images whose name matches `ghcr.io/kubewarden/*` with the keyless provided.
+1. GitHub actions. It will verify that all images were signed for a GitHub action with the `kubewarden` owner and in the repo `app-example`
 
-Another example that validates a GitHub action:
-
-```
+``` yaml
 signatures:
 - image: "ghcr.io/kubewarden/*"
+  owner: "kubewarden" #optional
   repo: "app-example"
-  owner: "kubewarden"
 ```
 
-This will validate the GitHub repo `app-example` for the `kubewarden` owner
-
-And a url prefix example:
+2. Keyless subject prefix. It will verify that the issuer is `https://token.actions.githubusercontent.com` and the subject starts with `https://github.com/kubewarden/app-example/.github/workflows/ci.yml@refs/tags/`
+   `url_prefix` is sanitized to prevent typosquatting.
 
-```
+``` yaml
 signatures:
 - image: "ghcr.io/kubewarden/*"
   keyless_prefix:
     - issuer: "https://token.actions.githubusercontent.com"
       url_prefix: "https://github.com/kubewarden/app-example/.github/workflows/ci.yml@refs/tags/"
-```
+``` 
+
+
+3. Keyless exact match. It will verify that the issuer is `https://token.actions.githubusercontent.com` and the subject is `kubewarden`. It will not modify the image with the digest.
 
-`url_prefix` will match all subjects that starts with `https://github.com/kubewarden/app-example/.github/workflows/ci.yml@refs/tags/`
-`url_prefix` is sanitized to prevent typosquatting.
+``` yaml
+modifyImagesWithDigest: false #optional. default is true
```

Comment:
This is the only time we show this settings inside of the example `yaml` files. I would make this example about mutation, hence set this value to `true`

Reply:
default is true. So all other examples (except this one) are modifying the image with the digest. That's why I thought it would be nice to give an example without mutation. This is explained [here](https://github.com/raulcabello/verify-image-signatures/blob/gha/README.md?plain=1#L9-L10)

Reply:
Sorry, I didn't read the README properly.